src/components/avatar.ts

```diff
@@ -1,28 +1,33 @@
 import { customElement, property } from "lit/decorators.js";
-import { html, type TemplateResult } from "lit";
-import { type HTMLImage, ImageDecoding } from "../shared/utils.ts";
+import type { TemplateResult } from "lit";
+import { html } from "lit";
+import type { HTMLImage } from "../shared/utils.ts";
+import { ImageDecoding } from "../shared/utils.ts";
 import { TailwindElement } from "../shared/tailwind.element.ts";
 
 @customElement("sandbox-avatar")
 export class SandboxAvatar extends TailwindElement {
   @property() image!: HTMLImage;
   @property() decoding: ImageDecoding = ImageDecoding.Async;
+  @property() size = 16;
 
   protected override render(): TemplateResult {
+    const cssSizeString = `h-${this.size} w-${this.size}`;
+
     return html`
       <div
-        class="h-16 w-16 rounded-full bg-white/90 p-0.5 shadow-lg shadow-zinc-800/5 ring-1 ring-zinc-900/5 backdrop-blur
+        class="${cssSizeString} w rounded-full bg-white/90 p-0.5 shadow-lg shadow-zinc-800/5 ring-1 ring-zinc-900/5 backdrop-blur
```

Comment:
This is the reason we need to preload the tailwind "w-" and "h-" classes.

We could add a property to the avatar component, that might be smarter?